src/pages/login.tsx

```diff
@@ -0,0 +1,16 @@
+import { Heading, HeadingLevels, Tile } from 'lbh-frontend-react';
+import Layout from '../components/ResidentLayout';
+import { ReactNode } from 'react';
+
+const Home = (): ReactNode => (
+  <Layout>
+    <Heading level={HeadingLevels.H1}>Please log in</Heading>
+    <Tile
+      link="https://auth.hackney.gov.uk/auth?redirect_uri=http://localhost:3000"
+      title="Log in with Google"
+    />
+    {/* <Link href={`https://auth.hackney.gov.uk/auth?redirect_uri=${redirect_uri}`} Login with google></Link> */}
+  </Layout>
+);
+
+export default Home;
```

Comment:
Not sure we need this page, as I think the homepage itself will be authenticated? I.e. the only public page we'll have will be the resident upload page I think

Reply:
we'll need a page with a "click here to log in" link though, won't we?

Reply:
Sure, that makes sense. Might as well be the homepage then?